cantools/database/can/formats/dbc.py

```diff
@@ -315,13 +316,15 @@ class DbcSpecifics(object):
     def __init__(self,
                  attributes=None,
                  attribute_definitions=None,
+                 environment_variables=None,
                  value_tables=None):
         self._attributes = attributes
         self._attribute_definitions = attribute_definitions
 
         if value_tables is None:
             value_tables = odict()
 
+        self._environment_variables = environment_variables
```

Comment:
Move to line 323, before all ``value_tables`` code.

Reply:
No problem